### PR TITLE
implement Button component

### LINKED ISSUE #8: Game flow control
screen shot 2017-09-20 at 7 21 38 pm

### PR DESCRIPTION
**Pull request by R1ZZU**
Closes #8

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { h } from 'preact';
+import Icon from 'react-fontawesome';
+
+const Button = ({ icon, text, disabled, onClick }) => (
+    <button onClick={onClick} disabled={disabled}>
+        { icon ? <Icon name={icon} /> : null }
```
Comment (kirilknysh) [inline src/components/Button/Button.js]: How to control icon position (left/right)?

Comment (R1ZZU) [reply src/components/Button/Button.js]: Do we need it?

Comment (kirilknysh) [reply src/components/Button/Button.js]: Check out your comment in a linked issue :)

File: src/components/Button/Button.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <button onClick={onClick} disabled={disabled}>
+        { iconLeft ? <Icon name={iconLeft} /> : null }
+        <span>{ text }</span>
+        { iconRight ? <Icon name={iconRight} /> : null }
+
+        <style jsx>{`
+        button {
+            color: #badece;
+            border: 2px solid #84b4aa;
+            border-radius: 1em;
```
Comment (DaQuirm) [inline src/components/Button/Button.js]: Maybe rethink `em`s, shouldn't be a big problem though

**Pull request merged**